code/datums/shuttles.dm

```diff
@@ -86,10 +86,16 @@
 /datum/map_template/shuttle/emergency/meteor
 	suffix = "meteor"
 	name = "Asteroid With Engines Strapped To It"
-	description = "A hollowed out asteroid with engines strapped to it. Due to its size and difficulty in steering it, this shuttle may damage the docking area."
+	description = "A hollowed out asteroid with engines strapped to it. Due to its size and poor navigation protocols, this 'shuttle' will damage the docking area."
 	admin_notes = "This shuttle will likely crush escape, killing anyone there."
 	credit_cost = -5000
 
+/datum/map_template/shuttle/emergency/meteor/prerequisites_met(mob/user)
+	if("emagged" in SSshuttle.shuttle_purchase_requirements_met)
+		return TRUE
+	to_chat(user, "<span class='notice'>The console blocks the purchase of that shuttle.</span>")
```

Comment:
The whole point of the proc existing is to make the messages not vague. It should be some lore-y explanation for the shuttle being present on the list but impossible to buy.

Reply:
Could make it as simple as "Central Command has deemed this shuttle too dangerous for general crew usage." or something similar.